Reorganize getting-started + added a new tab "Core Concepts" (#24)
php-etl/documentation: content/getting-started/environment-setup/_index.en.md

---

> title: "Environment Setup"

Reviewer: **🔨 issue:** Il manque le cas où on build sous ‏Docker.

Author: Effectivement, je le rajoute

date: 2023-02-14T14:53:57+01:00
draft: true
weight: 1
---

> Gyroscops is a command line application written in PHP. To be able to use the application correctly
> the application you must first have these 2 requirements.

Reviewer: ```suggestion
Gyroscops is a command line application written in PHP. To be able to use the application, you will need to fulfill the following requirements:
```

> ### PHP 8.0 or higher
> Before proceeding, make sure your computer has a PHP version higher than or equal to 8.0 installed.
> 
> For more information on how to install PHP, go to the [official documentation](https://www.php.net/manual/en/install.php).
> 

Reviewer: ```suggestion
### PHP 8.2 or higher
Before proceeding, make sure your computer has a PHP version higher than or equal to 8.2 installed.

For more information on how to install PHP, go to the [official documentation](https://www.php.net/manual/en/install.php).

```

---
title: "Environment Setup"
date: 2023-02-14T14:53:57+01:00
draft: true
weight: 1
---

Gyroscops is a command line application written in PHP. To be able to use the application, you will need to fulfill the following requirements:

### PHP 8.0 or higher
Before proceeding, make sure your computer has a PHP version higher than or equal to 8.0 installed.

For more information on how to install PHP, go to the [official documentation](https://www.php.net/manual/en/install.php).

### Composer 2

> After installing PHP, you must ensure that your Composer 2 is installed on your machine.

Reviewer 2: ```suggestion
After installing PHP, you must ensure that Composer 2 is installed on your machine.
```

Author: Modifié
